examples/state_teleportation.xbb

```diff
@@ -2,30 +2,19 @@
 name StateTeleportation
 # required blackbird API version it targets
 version 1.0
-# optional 'target' specifies the device the blackbird
```

Comment:
Why were all the lines in this file removed? Is the goal for this file to only serve as a subroutine in other programs?

Reply:
Exactly, I modified this to be a subroutine!